addons/calendar/wizard/mail_activity_schedule_views.xml

```diff
@@ -13,7 +13,7 @@
                   <attribute name="invisible">activity_category == 'meeting'</attribute>
             </xpath>
             <xpath expr="//button[@name='action_schedule_activities']" position="attributes">
-                  <attribute name="invisible">activity_category in ('meeting', 'phonecall') or id</attribute>
+                  <attribute name="invisible">has_error or activity_category == 'meeting' or id</attribute>
```

Comment:
Commit message doesn't explain why there's additional `has_error`, and the diff is unclear on why this is relevant. If this is necessary, could be nice to explain briefly why in commit message.